sdk/recoveryservices-backup/Azure.ResourceManager.RecoveryServicesBackup/api/Azure.ResourceManager.RecoveryServicesBackup.netstandard2.0.cs

```diff
@@ -1721,16 +1723,19 @@ public partial class IaasVmRestoreContent : Azure.ResourceManager.RecoveryServic
         public bool? DoesCreateNewCloudService { get { throw null; } set { } }
         public bool? DoesRestoreWithManagedDisks { get { throw null; } set { } }
         public Azure.ResourceManager.RecoveryServicesBackup.Models.VmEncryptionDetails EncryptionDetails { get { throw null; } set { } }
+        public Azure.ResourceManager.Resources.Models.ExtendedLocation ExtendedLocation { get { throw null; } set { } }
         public Azure.ResourceManager.RecoveryServicesBackup.Models.IdentityBasedRestoreDetails IdentityBasedRestoreDetails { get { throw null; } set { } }
         public Azure.ResourceManager.RecoveryServicesBackup.Models.BackupIdentityInfo IdentityInfo { get { throw null; } set { } }
         public bool? OriginalStorageAccountOption { get { throw null; } set { } }
         public string RecoveryPointId { get { throw null; } set { } }
         public Azure.ResourceManager.RecoveryServicesBackup.Models.FileShareRecoveryType? RecoveryType { get { throw null; } set { } }
         public Azure.Core.AzureLocation? Region { get { throw null; } set { } }
         public System.Collections.Generic.IList<int> RestoreDiskLunList { get { throw null; } }
+        public string SecuredVmOSDiskEncryptionSetId { get { throw null; } set { } }
```

Comment:
Could you confirm if the value of this property is always an arm resource id?
If so please change its format to `arm-id`

Reply:
updated as per suggestion